lib/src/model/language_feature.dart

```diff
@@ -4,12 +4,33 @@
 
 import 'package:dartdoc/src/render/language_feature_renderer.dart';
 
-const Map<String, String> _featureDescriptions = {};
-
-const Map<String, String> _featureUrls = {};
-
-/// An abstraction for a language feature; used to render tags to notify
-/// the user that the documentation should be specially interpreted.
+const Map<String, String> _featureDescriptions = {
+  'sealed': 'All direct subtypes must be defined in the same library.',
+  'abstract': 'This type can not be directly constructed.',
+  'base': 'This type can only be extended (not implemented or mixed in).',
```

Comment:
Probably in this context, better to say "This class can..." here, and in each of the ones below?

Reply:
`base` can apply to mixins too, but `class or mixin` isn't too long to say where it applies.  Will do.